cypher/cypher-docs/src/docs/dev/ql/administration/security/administration-security-administration-toc.asciidoc

```diff
@@ -12,4 +12,7 @@
 *** <<administration-security-administration-dbms-privileges-intro, DBMS privileges>>
```

Comment:
The different toc seems to not be completely synced for the DBMS privileges, please sync them all with how the section is structured